runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -235,17 +235,12 @@ void registerTable(@Nonnull String namespace,
              * Caller is opening a new table with a map of protobufFilename -> protobufFileDescriptor pairs
              * Here are the cases that can happen:
              * 1. No entry in TableRegistry -> New table
-             *  1.1: When inserting new protobufs into descriptor table, we find existing entries and they all match
-             *  1.2: When inserting new protobufs into descriptor table, we find mismatching [Schema Change!]
+             *  1.1: When inserting new protobufs into descriptor table, we find existing entries, we don't replace
+             *  (assumption is that protobuf definitions do not change outside of an upgrade/migration scenario)
              *
              * 2. Entry exists in TableRegistry -> Existing table re-opened:
-             *    2.1: Existing protobuf file list matches, newly inserted protobuf list
-             *    2.1.1: Each of Existing protobuf file descriptors match that in new file descriptor list
-             *    2.1.2: Existing protobuf file descriptor does not match that in new file descriptor![ SchemaChange!]
-             *
-             *    2.2: Existing protobuf file list does not match, newly opened protobuf list [big Schema Change]
-             *    2.2.1: Existing protobuf file list matches existing proto files
-             *    2.2.3: Existing protobuf file's descriptor mismatches with the incoming descriptor
+             *    2.1: Existing protoBuf file list is present, newly inserted protoBuf list (we don't check for equality/
+             *    match as the expectation is that schemas do not change outside of upgrade paths).
```

Comment:
Can you add a test that exercises the register table code path that verifies that the second registration is omitted.

Reply:
Added a test in `checkProtoDeduplication`
It compares the descriptor map's value before and after a schema update.